Packs/BluelivThreatContext/Integrations/BluelivThreatContext/BluelivThreatContext.yml

```diff
@@ -0,0 +1,769 @@
+      description: Malware Type
+      type: Unknown
+    - contextPath: malware.hasCandC
+      description: If there is a C&C associated
+      type: unknown
+    - contextPath: malware.memory
+      description: Malware memory
+      type: Unknown
+    - contextPath: malware.procMemory
+      description: Malware proc memory
+      type: Unknown
+    - contextPath: malware.analysisStatus
+      description: Malware analysis status
+      type: Unknown
+    - contextPath: malware.dropped
+      description: Malware dropped
+      type: Unknown
+    - contextPath: malware.buffers
+      description: Malware buffers
+      type: Unknown
+    - contextPath: malware.hasNetwork
+      description: If the malware has Network informations
+      type: Unknown
+    - contextPath: malware.risk
+      description: Malware associated risk
+      type: Unknown
+    - contextPath: malware.campaigns
+      description: Malware related campaigns
+      type: Unknown
+    - contextPath: malware.campaignIds
+      description: Malware related campaigns internal IDs
+      type: Unknown
+    - contextPath: malware.signatures
+      description: Malware signatures
+      type: Unknown
+    - contextPath: malware.sigantureIds
+      description: Malware sigantures internal IDs
+      type: Unknown
+    - contextPath: malware.threatActors
+      description: Malware threat actors
+      type: Unknown
+    - contextPath: malware.threatActorIds
+      description: Malware threat actors internal IDs
+      type: Unknown
+    - contextPath: malware.sources
+      description: Malware sources
+      type: Unknown
+    - contextPath: malware.sourceIds
+      description: Malware sources internal IDs
+      type: Unknown
+    - contextPath: malware.tags
+      description: Malware tags
+      type: Unknown
+    - contextPath: malware.tagIds
+      description: Malware tags internal IDs
+      type: Unknown
+    - contextPath: malware.crimeServers
+      description: Malware related crime servers
+      type: Unknown
+    - contextPath: malware.crimeServerIds
+      description: Malware crime servers internal IDs
+      type: Unknown
+    - contextPath: malware.fqdns
+      description: Malware FQDNs
+      type: Unknown
+    - contextPath: malware.fqdnIds
+      description: Malware FQDNs internal IDs
+      type: Unknown
+    - contextPath: malware.types
+      description: Malware types
+      type: Unknown
+    - contextPath: malware.typeIds
+      description: Malware types internal IDs
+      type: Unknown
+    - contextPath: malware.sparks
+      description: Malware sparks
+      type: Unknown
+    - contextPath: malware.sparkIds
+      description: Malware sparks internal IDs
+      type: Unknown
+    - contextPath: malware.ips
+      description: Malware IPs
+      type: Unknown
+    - contextPath: malware.ipIds
+      description: Malware IPs internal IDs
+      type: Unknown
+  - arguments:
+    - default: false
+      defaultValue: '0'
```

Comment:
no reason to have invalid default values.
please remove from all commands.
in the python code, you can check the dict returned by `demisto.args()` if they exist or not.